eslint.config.mjs

```diff
@@ -73,7 +73,7 @@ export default createConfigForNuxt({
         {
           argsIgnorePattern: '^_',
           ignoreRestSiblings: true,
-          varsIgnorePattern: '^_',
+          varsIgnorePattern: '',
```

Comment:
> The change to varsIgnorePattern from ^_ to empty string will now flag these previously allowed underscore-prefixed variables as unused, which might not be the intended behavior as they often serve as intentionally unused parameters or internal implementation details.

Do not confuse unused vars and unused args, unused vars are locally declared variables, if they are unused they cannot possibly affect the code as opposed to unused args which are arguments in a function where you might need some arguments but not all, or you might want to implicitly type a method in which case this could have an effect

Basically a change to 'argsIgnorePattern' might have unintended side effects, a change to 'varsIgnorePattern' will never have any side effect

Reply:
Would changing both `argsIgnorePattern` and `varsIgnorePattern` to `^_\w` be better?

Reply:
A few internal schemas declare noop functions with unused arguments just to expose implicit typing correctly, this could be fixed with explicit types

But I think it's better to just use the reportUsed feature for function arguments, which would prevent used arguments to start with an underscore, so that there is no more used variables with underscores that are used, which when the usage will be removed in the future will then properly report to eslint about the argument being unused